docs/deployment/config_descriptions.md

```diff
@@ -270,46 +270,32 @@ cluster are under 25%, then the cluster size will be reduced by 5%.
 #### Service AutoScaling
 
 Cumulus supports automatically scaling the number of tasks configured for an ECS
-service. The scaling of tasks is based on the `ActivityScheduleTime` metric,
-which measures how long (in milliseconds) an activity waited before being picked
-up for processing. If the average activity is waiting more than the configured
-`scaleOutActivityScheduleTime` time, then additional tasks will be added to the
-service. If the average activity is waiting less than the configured
-`scaleInActivityScheduleTime` time, then tasks will be removed from the service.
-Ideally, the average wait time for tasks should settle somewhere between
-`scaleInActivityScheduleTime` and `scaleOutActivityScheduleTime`.
+service.  The scaling of tasks is based on the specified metric.
```

Comment:
What are the different values that the "specified metric" could be? I think we should document that here along with maybe linking to some documentation for them.

Reply:
Also why you'd want to use different metrics.

Reply:
Added more descriptions here.